### PR TITLE
LRAC-13933 - [BUG] Segments criteria card are with extra spacing

### PR DESCRIPTION
**Pull request by ilzamcmed**
Jira ticket: https://issues.liferay.com/browse/LRAC-13933

Note: please see comment related with the first bug on the ticket

### CODE REVIEW

**Comment by ilzamcmed** [comment]
ci:test:sf

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Comment by ilzamcmed** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/workspaces/__tests__/__snapshots__/EmptyState.jsx.snap [hunk 1]
```diff
@@ -74,7 +74,7 @@ exports[`Workspace Empty State should render 1`] = `
           90 Day Full Feature Trial
         </p>
         <a
-          class="c-link link-secondary text-decoration-underline"
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/workspaces/__tests__/__snapshots__/EmptyState.jsx.snap:77]: Hi @ilzamcmed, why were these snapshots updated in this PR?

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/pages/__tests__/__snapshots__/Workspaces.jsx.snap [hunk 2]
```diff
@@ -313,7 +313,7 @@ exports[`Workspaces should render empty state 1`] = `
                   90 Day Full Feature Trial
                 </p>
                 <a
-                  class="c-link link-secondary text-decoration-underline"
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/pages/__tests__/__snapshots__/Workspaces.jsx.snap:316]: ^

**Review by interaminense — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/workspaces/__tests__/__snapshots__/EmptyState.jsx.snap (hunk 1 above)
Comment (ilzamcmed) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/workspaces/__tests__/__snapshots__/EmptyState.jsx.snap:77]: They were failing on master. Should I put them in another PR?

**Comment by interaminense** [comment]
ci:forward

**Comment by ilzamcmed** [comment]
ci:forward:force

**Pull request closed without merging**